docs/ota-client-guide/modules/ROOT/pages/add-board-class.adoc

```diff
@@ -0,0 +1,45 @@
+= Add a board class for new target board in meta-updater
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+Adding a board class for a new target board in meta-updater involves two main steps:
+
+. adding a new `sota_\{MACHINE}.bbclass` file for the board

```

Comment:
Needs to be made consistent with the second step. One's gerundive, the other's imperative. I think imperative would be better for both, but if you want to use the gerund, keep it consistent.

Reply:
Should be capitalized at the beginning of the sentence, too.